index.js

```diff
@@ -942,7 +977,7 @@
   //. > S.init([])
   //. Nothing()
   //. ```
-  S.init = R.ifElse(R.isEmpty, Nothing, R.compose(Just, R.init));
+  S.init = S.slice(0, -1);
```

Comment:
This might be somewhat easier to read(!):

```javascript
var at = S.at = R.curry(R.pipe(
    R.useWith(
      R.uncurryN(2, R.converge(
        S.slice,
        R.identity,
        R.ifElse(R.eq(-1), R.always(-0), R.inc))
      ),
      R.identity,
      R.identity),
    R.map(R.head)))
```